apepay/__init__.py

```diff
@@ -58,9 +58,44 @@ def normalize_address(cls, value: Any) -> AddressType:
 
     @property
     def contract(self) -> ContractInstance:
-        # TODO: Fix this
-        return self.project_manager.StreamManager.at(self.address)
-        # return self.chain_manager.contracts.instance_at(self.address)
+        try:
+            return self.chain_manager.contracts.instance_at(self.address)
```

Comment:
eventually we will want to include the contracts and reference the project via manifest, right?